### PR TITLE
wip: use parser plugins

### PR DESCRIPTION
**Pull request by noahnu**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     "@babel/preset-env": "^7.13.10",
     "@babel/preset-typescript": "^7.13.0",
     "@tophat/eslint-config": "^0.3.1",
-    "@types/babel__core": "^7.1.13",
+    "@types/babel__parser": "^7.1.1",
```
Comment (noahnu) [inline package.json:34]: pretty sure this doesn't have to be explicitly imported

File: src/getImportsFromWorkspaceMap.ts [hunk 2]
```diff
@@ -35,6 +35,26 @@ export default async function getImportsByWorkspaceMap(
     return importsMap
 }
 
+async function parseContent(
+    content: string,
+    { sourceFilename }: { sourceFilename: string },
+): Promise<ReturnType<typeof parse>> {
+    return parse(content, {
+        allowAwaitOutsideFunction: true,
+        errorRecovery: true, // skip errors if possible
```
Comment (noahnu) [inline src/getImportsFromWorkspaceMap.ts:44]: this makes it more of a best effort. ghost-imports is not meant to tell you if your code has a syntax error.. we're just trying to grab the imports slightly smarter than using regex/grep.

File: yarn.lock [hunk 3]
```diff
@@ -210,38 +210,37 @@ __metadata:
...
-  checksum: 5ca5eaa2658cc6738ce2810211084ec7cb2a1bbf9090d0ec1e9b9df1fd7786a0c4352f598eaafc682a722e7a0052afa73ee52d311b65544ca0e7f8597ed5242b
+    "@babel/types": ^7.13.12
+  checksum: 9abb5e3acb5630bf519b4205b7784947b64f93d573ed13579d894611392e48cac40b598f67b34c7b342fc6ac6d2262dcdecf125cac8806888328e914b2775c43
   languageName: node
   linkType: hard
 
 "@babel/helper-module-transforms@npm:^7.13.0":
-  version: 7.13.0
-  resolution: "@babel/helper-module-transforms@npm:7.13.0"
+  version: 7.13.12
```
Comment (noahnu) [inline yarn.lock:232]: updated babel as well

**Review by mcataford — APPROVED** [review]
LGTM!

**Pull request merged**